openedx/core/djangoapps/credentials/models.py

```diff
@@ -23,6 +23,9 @@ class Meta(object):
     CACHE_KEY = 'credentials.api.data'
 
     internal_service_url = models.URLField(verbose_name=_("Internal Service URL"))
+    version = models.PositiveIntegerField(
```

Comment:
See my previous comment. The odds of us needing to use two separate versions of the API are quite low. I can think of no reason to separate these URL components.